libs/cosmos-sdk/x/auth/ante/sigverify.go

```diff
@@ -202,10 +201,13 @@ func (svd SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simul
 		}
 
 		// retrieve signBytes of tx
-		signBytes := sigTx.GetSignBytes(ctx, signerAccs[i])
+		signBytes := sigTx.GetSignBytes(ctx, i, signerAccs[i])
+		if signBytes == nil {
+			return ctx, sdkerrors.Wrap(sdkerrors.ErrInvalidSignBytes, "signBytes is nil")
```

Comment:
this is a new case that returns an error if signByte == nil, it has some risks to smb